clojure_liberator/src/clojure_liberator/core.clj

```diff
@@ -21,6 +21,20 @@
    :type       type-name
    :attributes (dissoc value :id)})
 
+(defn- update-vals [map vals f]
+  (reduce #(update-in %1 [%2] f) map vals))
+
+(defn- parse-date [value]
```

Comment:
Haha yeah, my threshold is a bit lower than most, coming from Ruby. Clojure seems to be nearly as terse as Ruby though, which is awesome.